CHANGES.md

```diff
@@ -19,6 +19,7 @@
   * Fixed Karnaugh map color index bug.
   * Attribute sheet now honors application color theme.
   * Attribute sheet now displays HEX value of color properties.
+  * Added TTL 7487: 7487: 4-bit True/complement, zero/one
```

Comment:
Use full component name "7487: 4-bit True/complement, zero/one elements"

Reply:
Fixed in commit